src/agoradatatools/process.py

```diff
@@ -9,8 +8,12 @@
 import agoradatatools.etl.transform as transform
 import agoradatatools.etl.utils as utils
 from agoradatatools.errors import ADTDataProcessingError
+from agoradatatools.logs import log_time
+
+logger = logging.getLogger(__name__)
```

Comment:
@thomasyu888 What do you think about this way of initializing the `logger` in `process.py`, and then feeding it as an arg to `log_time`?

Reply:
Can you paste an example output of the execution?

Reply:
I don't love the "Uploading file to Synapse storage" interjections, but when I tried to silence them by doing `syn.store(..., silent=True)` in `load.py`, it also suppressed any logging output for that dataset after the `syn.store()` call.

There are a couple of examples in the description, but here is an entire output:
```
INFO | 2023-05-04 16:04:47 | Welcome, Brad Macdonald!

INFO | 2023-05-04 16:04:47 | Agora Data Tools processing has started
INFO | 2023-05-04 16:04:47 | Now processing genes_biodomains dataset
^C
Aborted!
(agora-data-tools) ➜  agora-data-tools git:(bwmac/AG-1041/add_timestamp_logging) adt test_config.yaml
Welcome, Brad Macdonald!

INFO | 2023-05-04 16:05:00 | Welcome, Brad Macdonald!

INFO | 2023-05-04 16:05:00 | Agora Data Tools processing has started
INFO | 2023-05-04 16:05:00 | Now processing genes_biodomains dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:12 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:05:13 | Processing complete for genes_biodomains dataset
INFO | 2023-05-04 16:05:13 | Elapsed time: 00:00:13 for genes_biodomains dataset
INFO | 2023-05-04 16:05:13 | Now processing neuropath_corr dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:15 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:05:16 | Processing complete for neuropath_corr dataset
INFO | 2023-05-04 16:05:16 | Elapsed time: 00:00:02 for neuropath_corr dataset
INFO | 2023-05-04 16:05:16 | Now processing proteomics dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:17 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 | INFO | 2023-05-04 16:05:18 | Processing complete for proteomics dataset
INFO | 2023-05-04 16:05:18 | Elapsed time: 00:00:01 for proteomics dataset
INFO | 2023-05-04 16:05:18 | Now processing proteomics_tmt dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:19 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 - INFO | 2023-05-04 16:05:19 | Processing complete for proteomics_tmt dataset
INFO | 2023-05-04 16:05:19 | Elapsed time: 00:00:01 for proteomics_tmt dataset
INFO | 2023-05-04 16:05:19 | Now processing target_exp_validation_harmonized dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:20 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:05:21 | Processing complete for target_exp_validation_harmonized dataset
INFO | 2023-05-04 16:05:21 | Elapsed time: 00:00:01 for target_exp_validation_harmonized dataset
INFO | 2023-05-04 16:05:21 | Now processing metabolomics dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:05:22 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 - INFO | 2023-05-04 16:05:23 | Processing complete for metabolomics dataset
INFO | 2023-05-04 16:05:23 | Elapsed time: 00:00:02 for metabolomics dataset
INFO | 2023-05-04 16:05:23 | Now processing gene_info dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:04 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:06:05 | Processing complete for gene_info dataset
INFO | 2023-05-04 16:06:05 | Elapsed time: 00:00:42 for gene_info dataset
INFO | 2023-05-04 16:06:05 | Now processing team_info dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:06 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:06:07 | Processing complete for team_info dataset
INFO | 2023-05-04 16:06:07 | Elapsed time: 00:00:01 for team_info dataset
INFO | 2023-05-04 16:06:07 | Now processing overall_scores dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:09 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 \ INFO | 2023-05-04 16:06:09 | Processing complete for overall_scores dataset
INFO | 2023-05-04 16:06:09 | Elapsed time: 00:00:02 for overall_scores dataset
INFO | 2023-05-04 16:06:09 | Now processing network dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:16 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 | INFO | 2023-05-04 16:06:17 | Processing complete for network dataset
INFO | 2023-05-04 16:06:17 | Elapsed time: 00:00:07 for network dataset
INFO | 2023-05-04 16:06:17 | Now processing rnaseq_differential_expression dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:46 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 | INFO | 2023-05-04 16:06:48 | Processing complete for rnaseq_differential_expression dataset
INFO | 2023-05-04 16:06:48 | Elapsed time: 00:00:30 for rnaseq_differential_expression dataset
INFO | 2023-05-04 16:06:48 | Now processing distribution_data dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:06:49 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:06:50 | Processing complete for distribution_data dataset
INFO | 2023-05-04 16:06:50 | Elapsed time: 00:00:02 for distribution_data dataset
INFO | 2023-05-04 16:06:50 | Now processing rna_distribution_data dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:07:06 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:07:07 | Processing complete for rna_distribution_data dataset
INFO | 2023-05-04 16:07:07 | Elapsed time: 00:00:17 for rna_distribution_data dataset
INFO | 2023-05-04 16:07:07 | Now processing proteomics_distribution_data dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:07:08 | 
##################################################
 Uploading file to Synapse storage 
##################################################

 / INFO | 2023-05-04 16:07:09 | Processing complete for proteomics_distribution_data dataset
INFO | 2023-05-04 16:07:09 | Elapsed time: 00:00:01 for proteomics_distribution_data dataset

##################################################
 Uploading file to Synapse storage 
##################################################

INFO | 2023-05-04 16:07:10 | 
##################################################
 Uploading file to Synapse storage 
##################################################

Uploading [####################]100.00%   250.0bytes/250.0bytes (313.2bytes/s) data_manifest.csv Done...
    INFO | 2023-05-04 16:07:12 | Agora Data Tools processing has completed
INFO | 2023-05-04 16:07:12 | Elapsed time: 00:02:11 for all data processing
```

Reply:
Interesting.. That is definitely a bug with the synapse client.  Suppressing logging in the client should not impact logging of other modules! I'll create a bug ticket to track this for the client.